### PR TITLE
[MGDOBR-1244] Exposed Ingress when a ManagedBridge Resource is created

### PR DESCRIPTION
**Pull request by robpblake**
[MGDOBR-1244](https://issues.redhat.com/browse/MGDOBR-1244)

This PR continues the re-use of the v1 data plane code and exposes an Ingress when a ManagedBridge resource is created.

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-XXX] - $clear_explanation_of_what_you_did"
- [x] The layers in the `kustomize` folder have been updated accordingly.
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

File: shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/controllers/ManagedBridgeController.java [hunk 1]
```diff
@@ -70,15 +81,14 @@ public UpdateControl<ManagedBridge> reconcile(ManagedBridge managedBridge, Conte
...
-        //
-        //        if (!status.isReady() && isTimedOut(status)) {
-        //            notifyManagerOfFailure(bridgeIngress,
-        //                                   new ProvisioningTimeOutException(String.format(ProvisioningTimeOutException.TIMEOUT_FAILURE_MESSAGE,
-        //                                                                                  bridgeIngress.getClass().getSimpleName(),
-        //                                                                                  bridgeIngress.getSpec().getId())));
-        //            status.markConditionFalse(ConditionTypeConstants.READY);
-        //            return UpdateControl.updateStatus(bridgeIngress);
-        //        }
+        if (!status.isReady() && isTimedOut(status)) {
```
[the remark this answers is not in the document]
Comment (manstis) [reply shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/controllers/ManagedBridgeController.java:84]: @robpblake IIRC @vaibhavjainwiz made comment that, in his opinion, timeout should be better handled in the Control Plane. I know for the initial `v2` implementation we're going "hard and fast" but IDK if there is a JIRA somewhere to record his proposal; for further consideration.

**Review by manstis — APPROVED** [review]
FWIW the code looks OK... IDK if what it _does_ is correct; but the code itself is fine.

File: shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/controllers/ManagedBridgeController.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (robpblake) [reply shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/controllers/ManagedBridgeController.java:84]: @jiripetrlik Done.

File: shard-operator-parent/shard-operator-v2/src/test/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+event-bridge.webhook.client-id=webhook-1
```
[the remark this answers is not in the document]
Comment (robpblake) [reply shard-operator-parent/shard-operator-v2/src/test/resources/application.properties:2]: @jiripetrlik Yes these are just test values that only work with the local setup of Keycloak. They have been in the repository for some time already.

**Review by jiripetrlik — APPROVED** [review]

**Pull request merged**